test/functional/test_framework/messages.py

```diff
@@ -849,11 +854,11 @@ def deserialize(self, f):
         self.nServices = struct.unpack("<Q", f.read(8))[0]
         self.nTime = struct.unpack("<q", f.read(8))[0]
         self.addrTo = CAddress()
-        self.addrTo.deserialize(f)
+        self.addrTo.deserialize(f, False)
```

Comment:
I think it's more clear to call with keyword arguments for boolean options: `deserialize(f, with_time=False)`. Same for invocations below.

Reply:
Agree; using a named argument would make this code more readable.